doc/howto/db-drivers.md

```diff
@@ -0,0 +1,28 @@
+# Install database drivers
```

Comment:
Do we need a separate guide for this? since DB drivers are already mentioned in the respective DB guides

Reply:
I feel like the guides contain so much information, that they'll confuse someone who just wants to know how to install the drivers. but we'll see this is more like an experiment, we can merge the content if we see it's too much duplication